apps/backend/src/websocket/server.py

```
@@ -0,0 +1,215 @@
+        ]
+    }
+
+
+# Health check endpoint
+@websocket_router.get("/health")
+async def websocket_health():
+    """WebSocket server health check"""
+    return {
+        "status": "healthy",
+        "service": "websocket_server"
+    }
```

Ruff (0.12.2)

No newline at end of file

Add trailing newline

(W292)